SharpPcap/PcapClock.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+namespace SharpPcap
+{
+    public class PcapClock
+    {
+        public string Name { get; }
+        public string Description { get; }
+        public TimestampSourceResolution Resolution { get; } // High, Low, Unknown
+
+        /// <summary>
+        /// If true, timestamps are synchronized with the host operating system
+        /// </summary>
+        public bool? Synchronized { get; }
+
+        /// <summary>
+        /// The type this clock maps back to
+        /// </summary>
+        public TimestampType TimestampType { get; }
+
+
+        public PcapClock(TimestampType timestampType, string name, string description)
+        {
+            Name = name;
+            Description = description;
+
+            Resolution = (timestampType == TimestampType.HostLowPrecision) ? TimestampSourceResolution.Low : TimestampSourceResolution.High;
+            Synchronized = ((timestampType == TimestampType.AdapterUnsynced) ||
```

Comment:
if TimestampType == TimestampType.Host then Synchronized = null (unknown)

Reply:
@kayoub5 that makes sense. So its unknown if we are synchronized or not if its host.